models/service.rb

```diff
@@ -27,15 +27,16 @@ class Service < Base
     # validates_numericality_of :exclusion_cost, allow_nil: true
     field :type, default: :service
 
-    field :first_possible_day, default: nil
-    field :last_possible_day, default: nil
+    field :first_possible_days, default: []
+    field :last_possible_days, default: []
+    field :first_possible_day, default: nil # this should be a VISIT attribute
```

Comment:
Did you mean, that each duplicated visit, should have only one first_possible_day, while the original on has an array ?

Reply:
In fact this should rather be on _visit_ model only but we don't have it yet. Avery visit would have only one first_possible day.

In the meantime, one service has as many first_possible_days as visits before expand, and one first_possibleçday after expand. We could indeed keep only _first_possible_days_ even after expand.